assets/js/base/components/chip/stories/removable-chip.stories.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { Story, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { RemovableChip, RemovableChipProps } from '../removable-chip';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/RemovableChip',
+	component: RemovableChip,
+} as Meta< RemovableChipProps >;
+
+const Template: Story< RemovableChipProps > = ( args ) => (
+	<RemovableChip { ...args } />
+);
+
+export const Default = Template.bind( {} );
+Default.args = {
+	text: 'Take me to the casino',
```

Comment:
You should provide a default element as an arg as well. Try to add an element string in the Storybook, when you click on the button it breaks.

You can provide non-story dependent args in the default export above. You basically do:

```
export default {
	title: 'WooCommerce Blocks/@base-components/RemovableChip',
	component: RemovableChip,
	args: {
	  element: 'li'
	}
} as Meta< RemovableChipProps >;
```

And all your stories will have that. It's pretty handy!